File: 1689.partitioning-into-minimum-number-of-deci-binary-numbers.py
#
# @lc app=leetcode id=1689 lang=python3
#
# [1689] Partitioning Into Minimum Number Of Deci-Binary Numbers
#

# @lc code=start
class Solution:
    def minPartitions(self, n: str) -> int:
        return int(sorted(list(n))[::-1][0])
# @lc code=end

File: 1689.partitioning-into-minimum-number-of-deci-binary-numbers.js
/*
 * @lc app=leetcode id=1689 lang=javascript
 *
 * [1689] Partitioning Into Minimum Number Of Deci-Binary Numbers
 */

// @lc code=start
/**
 * @param {string} n
 * @return {number}
 */
var minPartitions = function(n) {
  return +[...n].sort((a,b)=>b-a)[0]
};
// @lc code=end

